ui/packages/catalog-ui-search/src/main/webapp/component/query-time/query-time.view.js

```diff
@@ -55,6 +59,8 @@ module.exports = Marionette.LayoutView.extend({
     constructFilter: function () {
         var filters = [];
         var timeRange = this.basicTime.currentView.model.getValue()[0];
+        let timeSelection = this.basicTemporalSelections.currentView.model.getValue()[0];
+        timeSelection = (!Array.isArray(timeSelection) || !timeSelection.length) ? undefined : timeSelection;
```

Comment:
❓ Do we need the `isArray` check?

Reply:
Honestly, probably not unless for some reason the behavior changes in grabbing the selection from the dropdown. I'll remove it.